bullet_train/app/controllers/concerns/account/users/controller_base.rb

```diff
@@ -24,15 +24,15 @@ def edit
   def show
   end
 
-  def updating_password?
```

Comment:
I wonder if we can yank this? I'm guessing apps won't be relying on this since they may not be overriding `update` which uses it, but I don't know if that's a safe bet.

Reply:
🤔 Hmm... That's a good question, @kaspth. If we leave it in, should it do what it's always done and just check for `:password` or should we make it an alias for `updating_password_or_email?`?

Reply:
OK, instead of yanking the method I'm leaving it as it was, but adding a deprecation message. If someone is using it they'll see something like this in their console:

```
DEPRECATION WARNING: #updating_password? is deprecated. \
  Use #updating_password_or_email? instead. \
  (called from block in update at /Users/jgreen/projects/bullet-train-co/bullet_train-core/bullet_train/app/controllers/concerns/account/users/controller_base.rb:43)
```